fastlane/Fastfile

```diff
@@ -65,7 +165,4 @@ before_all do |lane|
   unless is_ci || File.file?(USER_ENV_FILE_PATH)
     UI.user_error!("#{USER_ENV_FILE_PATH} not found: Please copy fastlane/env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values")
   end
-
-  # Ensure we use the latest version of the toolkit
-  check_for_toolkit_updates unless is_ci || ENV['FASTLANE_SKIP_TOOLKIT_UPDATE_CHECK']
```

Comment:
My thinking was that it's better to update the Release Toolkit earlier in the code freeze than later so that any changes can be tested. So having it check before every lane is run doesn't seem efficient. But I see your point and will add it back to `before_all`.

Reply:
If we want to go with that route, I think `complete_code_freeze` would make the most sense. While running that lane, we are already in the `release` branch and it's a decent time to do the update. However, if we want to do that, I think it should be done for all apps after consensus amongst the team. We should also separate it into its own PR since it is not related to this one.

Reply:
I moved it back to `before_all` in https://github.com/wordpress-mobile/WordPress-Android/commit/6ab18efe9261a63b77766614c06c3bc661a697f6 so we can discuss it with the team before making any changes here.